dashboard/db/migrate/20170330214255_create_pd_program_registration.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePdProgramRegistration < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_facilitator_program_registrations do |t|
+      t.references :user, index: true
+      t.text :form_data
+
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
In my (possibly overcareful) world, the migration reaches all environments, then the model reaches all environments, and then the usage of the model reaches all environments.

If the migration and model (its usage) are PRed together, then if the migration fails (alternately another (earlier) migration in that deploy), the model refers to a non-existent table.